lib/matplotlib/tests/test_animation.py

```diff
@@ -75,6 +76,14 @@ def test_null_movie_writer():
     assert writer._count == num_frames
 
 
+def test_animation_delete():
+    anim = make_animation(frames=5)
+
+    with pytest.warns(Warning, match='Animation was garbage-collected'):
```

Comment:
```suggestion
    with pytest.warns(Warning, match='Animation was deleted'):
```

Reply:
To fix CI. Anybody can merge after CI pass.

Reply:
Oops, fixed.